backend_nodejs/.env

```diff
@@ -1,2 +1,13 @@
 PORT=3000
-# DATABASE_URL=mongodb://localhost:27017/yourdb
\ No newline at end of file
+
+DB_USER=postgres
+DB_PASSWORD=Notimportant.1
+DB_HOST=localhost
+DB_PORT=5432
+DB_DATABASE=arguments_mdg_db
```

Comment:
better store these in enironment variables locally on your machine than pushing them to git